app/poros/bio.rb

```diff
@@ -1,25 +1,33 @@
 class Bio
-  attr_reader :ratings, :name, :state, :age, :district, :photo, :gender, :years_in_office, :next_election, :type, :party
+  attr_reader :ratings, :name, :state, :age, :district, :photo, :gender, :years_in_office, :next_election, :congressional_type, :party
   def initialize(data)
       @name = data[:candidate][:firstName] + " " + data[:candidate][:lastName]
-      @type = data[:office][:title]
-      @state = @type == "Senator" ? data[:office][:stateId] : data[:office][:stateId]
+      @congressional_type = find_data(data[:office], :title)
```

Comment:
'find_data' is a really interesting solution to the problem of Irregular Json. Nice work.